packages/devkit/src/utils/wrap-rule.ts

```diff
@@ -0,0 +1,7 @@
+import { Tree } from '@nrwl/tao/src/shared/tree';
+
+export function wrapRule(callback: (...args) => any) {
```

Comment:
I created this simple function because when we generate new plugins (or any workspace) there's linting that does not like TypeScript's `any`. Rather than have to check if we're using tslint or eslint, we have this wrapper function